### PR TITLE
Provide a basic guide for using the meta-everest layer

### PR DESCRIPTION
**Pull request by drmrd**

### CODE REVIEW

File: docs/tutorials/how_to_yocto/index.rst [hunk 1]
```diff
@@ -2,25 +2,24 @@
...
+Steps to build EVerest in a Yocto image
+=======================================
 
 #. Set up a Yocto development environment.
 
-   -  The `Yocto Project Quick
-      Build <https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html>`__
-      guide is an excellent place to start.
+   .. TIP::
+      The `Yocto Project Quick Build`_ guide is an excellent place to start.
```
Comment (christopher-davis-afs) [inline docs/tutorials/how_to_yocto/index.rst:11]: Something we ran into while trying to run these steps is confusion on where we were supposed to diverge from the Yocto Project Quick Build. There was an impression that we weren't supposed to follow it until the end, but instead follow it partially and switch over to this document.

I feel like we should include the relevant steps from the Yocto documentation here, and point to the documentation as an additional point of reference. That way this document becomes a self-contained tutorial on using meta-everest.

**Review by folkengine — APPROVED** [review]
Looking forward to really kicking the tires on this. It opens up so many possibilities.

File: docs/tutorials/how_to_yocto/index.rst [hunk 2]
```diff
@@ -148,11 +145,43 @@ Steps to building EVerest in a Yocto image
...
+.. |meta-everest layer| replace:: ``meta-everest`` layer
+.. _meta-everest layer: https://github.com/EVerest/meta-everest
+
+
+Integrating Third-Party Modules
+===============================
+
+Sometimes you want to use EVerest to control hardware that lacks a board support module in EVerest.
+Sometimes you want to incorporate value-added services into your charger to distinguish yourself from the competition.
+And sometimes you just want to abuse EVerest's flexibility and use the platform to power a digital picture frame.
```
Comment (ghost) [inline docs/tutorials/how_to_yocto/index.rst:168]: 🤣

**Review by ghost — COMMENTED** [review]
The yocto tone is much more official than the Integrating Third-Party Modules section.

**Pull request merged**